src/steps/generic.rs

```diff
@@ -85,6 +85,18 @@ pub fn run_sheldon(ctx: &ExecutionContext) -> Result<()> {
     target_os = "netbsd",
     target_os = "dragonfly"
 )))]
+pub fn run_vscode(run_type: RunType) -> Result<()> {
+    let _vscode = utils::require("code")?;
+
+    print_separator("Visual Studio Code");
+
+    run_type
+        .execute(shell())
+        .arg("-c")
+        .arg("for ext in $(code --list-extensions); do code --force --install-extension \"$ext\"; done")
```

Comment:
Thanks, but we need to avoid using a Unix shell because it won't work in Windows

Reply:
What's the best practice here? Implementing the needed loop in rust I assume?